incubator/hnc/pkg/controllers/hierarchy_controller.go

```diff
@@ -468,6 +465,7 @@ func (r *HierarchyReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				}},
 			}
 		})
+	r.namespaceLocks = map[string]*sync.Mutex{}
```

Comment:
This looks like it crept in from one of the squashed commits? I think this shouldn't be here.

Reply:
yes I was working on it. It should be good now!